source/includes/example-connect-onprem-to-atlas.rst

```diff
@@ -0,0 +1,79 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
```

Comment:
Given that this occurs at the start of a new section, replace `cluster0` and `cluster1` with
```
:setting:`cluster0`
:setting:`cluster1`
```
to provide the appropriate links.